cpp/test/Glacier2/override/test.py

```diff
@@ -14,8 +14,6 @@
     "Ice.Warn.Dispatch": "0",
     "Ice.Warn.Connections": "0",
     "Ice.TCP.SndSize": "100000",
-    "Ice.ThreadPool.Server.Serialize": "1",
```

Comment:
More useless Serialize?

Reply:
It's most likely here because the Glacier2 router is started with `--Ice.ThreadPool.Server.SizeMax=3` (like any Ice server from the test suite). So it's useful to ensure that requests from the client are forwared in the same order... `MaxDispatches=1` can be used instead.

Reply:
Note that we had both Ice.ThreadPool.Server.Serialize AND Ice.ThreadPool.Client.Serialize.

Ice.ThreadPool.Server.Serialize => "serializes" regular requests received by Glacier2 router (on its two OAs, since they don't configure their own thread pools).

Ice.ThreadPool.Client.Serialize => "serializes" bi-dir requests?

Since the test passes even though I didn't configure anything, it's not a good test.